helm/charts/hydra-maester/README.md

```diff
@@ -6,34 +6,35 @@ A Helm chart for Kubernetes
 
 ## Values
 
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| adminService | object | `{"name":null,"port":null}` | Connection data to admin service of Hydra |
-| adminService.name | string | `nil` | Service name |
-| adminService.port | string | `nil` | Service port |
-| affinity | object | `{}` | Configure node affinity |
-| deployment.args | object | `{"syncPeriod":""}` | Arguments to be passed to the program |
-| deployment.args.syncPeriod | string | `""` | The minimum frequency at which watched resources are reconciled |
-| deployment.automountServiceAccountToken | bool | `true` |  |
-| deployment.extraAnnotations | object | `{}` | Deployment level extra annotations |
-| deployment.extraLabels | object | `{}` | Deployment level extra labels |
-| deployment.nodeSelector | object | `{}` | Node labels for pod assignment. |
-| deployment.podMetadata | object | `{"annotations":{},"labels":{}}` | Specify pod metadata, this metadata is added directly to the pod, and not higher objects |
-| deployment.podMetadata.annotations | object | `{}` | Extra pod level annotations |
-| deployment.podMetadata.labels | object | `{}` | Extra pod level labels |
-| deployment.resources | object | `{}` |  |
-| deployment.securityContext | object | `{"allowPrivilegeEscalation":false,"capabilities":{"drop":["ALL"]},"privileged":false,"readOnlyRootFilesystem":true,"runAsNonRoot":true,"runAsUser":1000}` | Default security context |
-| deployment.tolerations | list | `[]` | Configure node tolerations. |
-| enabledNamespaces | list | `[]` | The Controller have CREATE and READ access to all Secrets in the namespaces listed below. |
-| forwardedProto | string | `nil` |  |
-| image.pullPolicy | string | `"IfNotPresent"` | Image pull policy |
-| image.repository | string | `"oryd/hydra-maester"` | Ory Hydra-maester image |
-| image.tag | string | `"v0.0.25"` | Ory Hydra-maester version |
-| imagePullSecrets | list | `[]` | Image pull secrets |
-| pdb | object | `{"enabled":false,"spec":{"minAvailable":1}}` | PodDistributionBudget configuration |
-| priorityClassName | string | `""` | Pod priority |
-| replicaCount | int | `1` | Number of replicas in deployment |
-| singleNamespaceMode | bool | `false` | Single namespace mode. If enabled the controller will watch for resources only from namespace it is deployed in, ignoring others |
+| Key                                     | Type   | Default                                                                                                                                                    | Description                                                                                                                      |
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞